packages/cli/src/utils/units.ts

```diff
@@ -1,5 +1,22 @@
-import { ZWeb3 } from 'zos-lib';
+import BN from 'bignumber.js';
+import web3Utils from 'web3-utils';
 
 export function isValidUnit(unit: string): boolean {
-  return ZWeb3.getUnits().includes(unit.toLowerCase());
+  return Object.keys(web3Utils.unitMap).includes(unit.toLowerCase());
+}
+
+export function prettifyTokenAmount(amount: string, decimals?: string, symbol?: string): string {
+  const prettifiedAmount = decimals
+    ? new BN(amount).dividedBy(new BN(10).exponentiatedBy(decimals)).toFormat()
```

Comment:
```suggestion
    ? new BN(amount).shiftedBy(-decimals).toFormat()
```
See http://mikemcl.github.io/bignumber.js/#shift